src/devices/Pn532/Pn532.cs

```diff
@@ -808,6 +808,14 @@ public override int Transceive(byte targetNumber, ReadOnlySpan<byte> dataToSend,
             return -1;
         }
 
+        /// <inheritdoc/>
+        public override bool ReselectTarget(byte targetNumber)
+        {
+            var ret = DeselectTarget(targetNumber);
```

Comment:
is deselecting necessary for this to succeed? Is it possible there is nothing to deselect?

Reply:
In theory no. It's not. But the select is. Let me adjust it. I did in the Pn5180 for the B card.